test/unittests/test_format_en.py

```diff
@@ -117,6 +116,55 @@ def test_no_speech(self):
 
 
 class TestPronounceNumber(unittest.TestCase):
+    def test_pronounce_digits_float(self):
+        self.assertEqual(pronounce_digits(0.5), "zero point five")
+        self.assertEqual(pronounce_digits(1.1235), "one point one two")
+        self.assertEqual(pronounce_digits(10.999999), "ten point nine nine")
+        self.assertEqual(pronounce_digits(10.999999, places=0), "ten")
```

Comment:
These aren't correctly rounded; I think this should either treat `places` as "significant figures", or specifically note in the docstring that `places` are not sig figs and decimals are truncated as opposed to rounded

Reply:
maybe an extra flag for rounding or not defaulting to True?

Reply:
I suspect its parsing as a string instead of using `round()`

Reply:
The method is a formating method should not modify input, imho not rounding is the expected behaviour and rounding should be done before calling method if needed or via optional flag. This is reading digits not pronouncing the number, the rounding feels application specific even if I can't come up with a good use case where we don't want it....